src/program.js

```diff
@@ -246,6 +254,13 @@ Example: $0 --help run.
       describe: 'Show verbose output',
       type: 'boolean',
     },
+    'config': {
+      alias: 'c',
+      describe: 'Supply a custom configuration file',
+      type: 'string',
+      requiresArg: true,
+      demand: false,
+    },
```

Comment:
You can also remove this code for now and do it later when you work on https://github.com/mozilla/web-ext/issues/775